### ISSUE TITLE
<Content Review> from <Benefits Dependent Experience>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#65553]
## About your team

- Team name: Benefits Dependent Experience
- OCTO-DE product owner: Emily Theis
- Product manager: Laura Steele
- Designer: Ajia Wallace and Julie Pedtke
- Accessibility specialist: none
- Content writer: Ajia Wallace
- Slack channel: benefits-ce-dependents_exp
 
## CAIA Support Request
We are making some changes to the contact information page on the 686c form and we would like CAIA to review. The changes include:
- A new header to clarify the form is asking for the Veteran's mailing address
- Two new error messages to mitigate address entry errors

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [x] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [x] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** to make the requested content changes in the 686c form
- **Provide date if available**: Start on 9/20/2023

#### Timing for your next step: 
  - [ ] Next week
  - [x] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [x] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users:** before the end of September
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: 
- **Collab. Cycle Ticket Opened (date)**: xx/xx/xxxx
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ebenefits/view-update-dependents/README.md
- **User flows**: 
- **Prototype or mockups**: https://www.sketch.com/s/870d13d5-a465-48a3-acc2-32682ba8f780/p/AFA43D59-B983-46B6-9702-F37DDB5ABF6D/canvas#Comment
- **If this is in Sketch, which version or specific page should we focus on?** Solution 3
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**: https://app.zenhub.com/workspaces/benefits-dependents-experience-6459139e7d64bf0015f6af13/issues/gh/department-of-veterans-affairs/va.gov-team/63545
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [x] Julie Strotham

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.
- [ ] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by steele-lm** [comment]
@juliepedtke

**Comment by juliepedtke** [comment]
Hi @strelichl @coforma-jamie Would you be able to provide an estimated timeline for when we can expect comments back? Thanks!

**Comment by coforma-jamie** [comment]
## CAIA accessibility review
Things are looking good from my end. You did a nice job adjusting the heading levels and putting email/phone on a separate page!

**Note:** There's a known issue around the form header, but that's a larger issue that's being worked on. (In short, the `<h1>` being repeated on every step of the form is an accessibility issue.) Check out [this epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/55121) and a [mockup](https://staging.va.gov/authorization-to-disclose-alternate/introduction) that's currently being tested. But this won't be implemented for awhile, so progress as you are for now!

**Comment by juliepedtke** [comment]
Thanks @coforma-jamie !

My understanding is that we are still waiting on content feedback, is there an estimate for when we can expect that @strelichl @laurwill @sara-amanda ? I'm not sure who will be working on it.

**Comment by coforma-terry** [comment]
Hey @juliepedtke - chasing up the content side of this for you now - will get back to you asap :)

**Comment by laurwill** [comment]
Hi @juliepedtke , I've added content suggestions in comments on the Sketch file. Let me know if you have any questions!

Also noting a larger item to discuss here: per our conversation in this [email review ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65222#event-10320260730), the "Add or remove dependents" form changes dependent info on disability, pension, and VA DIC benefits — not only on disability. The Sketch screens currently say "Add or remove dependents on your disability benefits" in the H1. 

If this form does change dependent information on pension and DIC in addition to disability, you'll need to make global changes to the form to reflect that. Happy to discuss that further with you all!

Our static page "View or change dependents" also has disability in the H1. This is an older page that also has some outdated information on it. We'll open a ticket to update that page for accuracy, and coordinate with your team to make sure our changes align with work you're doing in the online forms and tools.

cc @strelichl

**Comment by ajialeilani** [comment]
Hi @laurwill - thank you for your feedback. I've updated the [Sketch file](https://www.sketch.com/s/870d13d5-a465-48a3-acc2-32682ba8f780) based on your comments. I think my only outstanding question was to confirm the spacing/format of the updated "Need help?" content. Let me know if you see further changes needed.

**Comment by ajialeilani** [comment]
Hi @laurwill @coforma-jamie @coforma-terry - We're looking for some specific messaging to address the following rule: 
{    if mlty_postal_type_cd is AA, first three characters of zipCode should be 340 else reject transaction if mlty_postal_type_cd is AE, zipCode range should be (090-099) else reject transaction if mlty_postal_type_cd is AP, zipCode range should be (962-966) else reject transaction  }

Per @jstrothman 's suggestion, we're using "Because your address is outside the United States on a military base, please provide an APO/FPO/DPO postal code."

Here's a mockup of this messaging that I added to the [Sketch file](https://www.sketch.com/s/870d13d5-a465-48a3-acc2-32682ba8f780). 

Can we have this update reviewed as a part of this ticket? 
![Uploading DPO field.png…]()

**Comment by coforma-jamie** [comment]
Hi @steele-lm - do you know if this ticket still requires CAIA review, or did y'all make updates after collaboration cycle touchpoints and this is now out of date? Thanks!

**Comment by steele-lm** [comment]
@coforma-jamie It's out of date now that we're leveraging the collab cycle. Staging review for the address component changes is scheduled for 12/8!

**Comment by laurwill** [comment]
Hi @steele-lm @ajialeilani , this mailing address pattern has come up on another project, and I wanted to confirm the outcome from your collab cycle for this. 

Could you drop a link to the final version? And if it's different from the design system pattern, did you submit and experimental design? Thanks very much!

**Comment by ajialeilani** [comment]
Hi @laurwill—from what I understand, the design under [Solution 3 in this mockup](https://www.sketch.com/s/870d13d5-a465-48a3-acc2-32682ba8f780/v/ll8Vjd/p/AFA43D59-B983-46B6-9702-F37DDB5ABF6D/canvas/) went to production. 

@steele-lm @evansmith let me know if y'all saw something different.

**Comment by steele-lm** [comment]
@laurwill We did implement a [short-term fix](https://www.sketch.com/s/870d13d5-a465-48a3-acc2-32682ba8f780/p/AFA43D59-B983-46B6-9702-F37DDB5ABF6D/canvas#Comment) for this address issue! Here is the [Collab Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68019) ticket if that's helpful. We did not submit a experimental design for this primarily because there is (was?) a broader effort underway to redesign the address component as a whole and implement a better holistic solution. I know Ryan Thurwell was working on it for a while, and Shannon Ford was also involved. We recently reviewed our error logs and this address change has stopped users from submitting claims with domestic addresses when they've indicated they live on a military base overseas.
